provisioning/tasks/www.yml

```diff
@@ -88,7 +88,7 @@
 
 - name: Copy Nginx vhosts into place.
   template:
-    src: ../templates/nginx-vhost.conf.j2
+    src: "{{ nginx_vhost_template }}"
     dest: "{{ nginx_vhost_path }}/{{ item.server_name.split(' ')[0] }}.conf"
     force: yes
     owner: root
```

Comment:
Yes, in fact that's how I do it for my main Drupal multisite server (home of www.midwesternmac.com, www.jeffgeerling.com, www.beforeidostl.com among others...). I just never pulled that back into the Nginx role.

It makes things easier to manage, but I've been selfish and never contributed it back! :P